heroic-component/src/main/java/com/spotify/heroic/shell/ShellIO.java

```diff
@@ -43,19 +42,17 @@
      * Open a file for reading.
      *
      * @param path Path of file to open.
-     * @param options Options when opening file.
      * @return An InputStream associated with the open file.
      */
-    InputStream newInputStream(Path path, StandardOpenOption... options) throws IOException;
```

Comment:
What were these options and how/do they still work?

Reply:
The `StandardOpenOption` enumerates the POSIX params for opening a file - read, write, append, etc.

AFAICT the only thing that was ever passed in was `[READ]` so I removed it as an option.